Glideslope.json

```diff
@@ -14800,11 +14800,11 @@
    "identifier": "IFAR",
    "associatedRunwayNumber": "36",
    "frequency": 110300,
-   "elevation": 897,
-   "latitude": 46.911058,
-   "longitude": -96.814728,
-   "bearing": 0.942,
-   "glideslope": 3.00000471,
+   "elevation": 902,
+   "latitude": 46.908192,
+   "longitude": -96.816403,
+   "bearing": 0.47,
+   "glideslope": 3.000047,
```

Comment:
This is a confusing one, but this should be 3.0000047 because the bearing is technically 000.47